doc/source/conf.py

```diff
@@ -53,12 +54,12 @@
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
-version_info = pbr.version.VersionInfo(project)
+version_info = packaging.version.parse(importlib.metadata.version(project))
 
 # The short X.Y version.
-version = version_info.version_string()
+version = version_info.base_version
 # The full version, including alpha/beta/rc tags.
-release = version_info.release_string()
+release = version_info.public
```

Comment:
For the maintainer, you might also consider using `jaraco.packaging`, which provides something similar as a sphinx plugin. See [jaraco/skeleton](/jaraco/skeleton) for how that's configured.